include/tvm/expr.h

```diff
@@ -115,6 +115,43 @@ class Var : public PrimExpr {
   using ContainerType = VarNode;
 };
 
+class ShapeVar;
+/*!
+ * \brief A variable node represent a tensor shape size,
+ * whose value must be non-negative.
+ */
+class ShapeVarNode : public VarNode {
+ public:
+  static ShapeVar make(DataType dtype, std::string name_hint);
```

Comment:
use constructor in ShapeVar, as per https://github.com/apache/incubator-tvm/issues/4648